src/reverse/RTTIMapper.cpp

```diff
@@ -130,55 +141,60 @@ void RTTIMapper::RegisterSimpleTypes(sol::state& aLuaState, sol::table& aLuaGlob
         data.value = aVariant.GetDataPtr();
 
         auto luaLock = m_lua.Lock();
-        
+
```

Comment:
Remove this ?

Reply:
i dont see an issue with removing trailing whitespace honestly, unless it makes changes unreadable